src/dfx/src/commands/ledger/mod.rs

```diff
@@ -135,7 +136,7 @@ pub async fn transfer(
                     fee,
                     from_subaccount,
                     to,
-                    created_at_time: Some(TimeStamp { timestamp_nanos }),
+                    created_at_time: Some(TimeStamp { timestamp_nanos: created_at_time.unwrap_or(timestamp_nanos) }),
```

Comment:
Recommend setting this at the beginning of this method, `let timestamp_nanos = `, to make it clear that the SystemTime::now()... expression isn't used at all if created_at_time is Some.

Reply:
Sure! See https://github.com/dfinity/sdk/pull/3018/commits/803dfdadc37efea6a9232fbec52f60d2af93878e.

Reply:
Do you rather a match statement for clarity or is unwrap_or() good?